dashboard/modules/node/node_head.py

```diff
@@ -6,7 +6,8 @@
 from aioredis.pubsub import Receiver
 from grpc.experimental import aio as aiogrpc
 
-import ray.gcs_utils
+import ray._private.utils
```

Comment:
Is this used?

Reply:
Hmm not sure how this got here 🤔 I'll remove it

Reply:
Yep it's used below, [here](https://github.com/ray-project/ray/blob/e988fa9470cbd76db8858783eb871f899ebc870c/dashboard/modules/node/node_head.py#L249). Not sure how it wasn't failing before.